src/Kernels/DynamicRupture.h

```diff
@@ -66,6 +66,7 @@ class seissol::kernels::DynamicRupture {
   private:
     dynamicRupture::kernel::evaluateAndRotateQAtInterpolationPoints m_krnlPrototype;
     kernels::Time m_timeKernel;
+    seissol::initializer::parameters::DamagedElasticParameters* damagedElasticParameters;
```

Comment:
I'd add the `m_` prefix here to be consistent with the rest of the code in this file ....